.github/workflows/PKGBUILD.template

```diff
@@ -0,0 +1,42 @@
+# Maintainer: peippo <christoph+aur@christophfink.com>
+
+pkgname=python-r5py
+_name=${pkgname#python-}
+pkgdesc="Rapid Realistic Routing with R5 in Python"
+url="https://github.com/r5py/r5py"
+
+pkgver=
+pkgrel=1
+
+arch=("any")
+license=("GPL")
```

Comment:
Is it possible to have the dual license here as well?

Reply:
I wasn’t sure, and then I forgot to look it up, but yes!